GitCommands/Git/Executable.cs

```diff
@@ -45,6 +46,50 @@ public Executable(Func<string> fileNameProvider, string workingDir = "", string
             return new ProcessWrapper(fileName, _prefixArguments, args, _workingDir, createWindow, redirectInput, redirectOutput, outputEncoding, useShellExecute, throwOnErrorExit);
         }
 
+        /// <summary>
+        /// Open a file with its associated default application.
+        /// </summary>
+        /// <param name="filePath">Pathname of the file to open.</param>
+        public static void Open(string filePath)
```

Comment:
But these utility functions should not be merged into the `Executable` class.
Rather make `OsShellUtil` available for plugins, please.

Reply:
I tend to agree with @mstv, I had the same first reaction that we shouldn't do this...

Reply:
Done.